src/lib/openapi/spec/client-feature-schema.ts

```diff
@@ -9,55 +9,67 @@ export const clientFeatureSchema = {
     $id: '#/components/schemas/clientFeatureSchema',
     type: 'object',
     required: ['name', 'enabled'],
+    description:
+        'Feature toggle configuration used by SDKs to evaluate state of a toggle',
     additionalProperties: false,
     properties: {
         name: {
             type: 'string',
+            description:
+                'The unique name of a feature toggle. Is validated to be URL safe on creation',
+            example: 'new.payment.flow.stripe',
         },
         type: {
             type: 'string',
+            description: 'What kind of feature flag is this',
+            example: 'release',
         },
         description: {
             type: 'string',
+            description: 'A description of the toggle',
             nullable: true,
-        },
-        createdAt: {
-            type: 'string',
-            format: 'date-time',
-            nullable: true,
-        },
-        lastSeenAt: {
-            type: 'string',
-            format: 'date-time',
-            nullable: true,
+            example: 'No variants here',
         },
         enabled: {
             type: 'boolean',
+            description:
+                'Whether the feature flag is enabled for the current API key or not. This is ANDed with the evaluation results of the strategies list, so if this is false, the evaluation result will always be false',
```

Comment:
Should this be 'environment' or 'API key'? 

```suggestion
                'Whether the feature flag is enabled for the current environment or not. This is ANDed with the evaluation results of the strategies list, so if this is false, the evaluation result will always be false',
```